tests/unit/AGMDate.spec.ts

```diff
@@ -13,7 +13,7 @@ Vue.use(Vuetify)
 Vue.use(Vuelidate)
 
 const vuetify = new Vuetify({})
-const store = getVuexStore()
+const store = getVuexStore() as any // remove typings for unit tests
```

Comment:
... Because we want the tests to be fragile (eg, use strings instead of types in tests so that we end up testing the types too).